src/bubblewrap.py

```diff
@@ -3,15 +3,16 @@ class Bubbles():
     def __init__(self):
         pass
 
-    def get_rich_bubble(self, txt, bg_color="blue", fg_color="white", rich=True):
-        circle_style = f"[{bg_color}]"
-        circle_close = f"{circle_style[:1]}/{circle_style[1:]}"
-        body_open = f"[{fg_color} on {bg_color}]"
-        body_close = f"{body_open[:1]}/{body_open[1:]}"
-        return f"{circle_style}{circle_close}{body_open}{txt}{body_close}{circle_style}{circle_close}"
+    def get_rich_bubble(self, txt, bg_color="blue",
+                        fg_color="white", rich=True):
+        c_style = f"[{bg_color}]"
+        c_close = f"{c_style[:1]}/{c_style[1:]}"
+        b_open = f"[{fg_color} on {bg_color}]"
+        b_close = f"{b_open[:1]}/{b_open[1:]}"
```

Comment:
Could you please revert these to `circle_style/close` and `body_open/close` i know it's longer but i like i more because it makes it very clear what each variable is for

Reply:
okay. Will do it